### DISCUSSION TITLE
Unwritten rules that should be made clearer

### DISCUSSION THREAD

**Discussion by EmmaDawsonDev** [EddieHubCommunity/support#3985] [Ideas]
What are the unwritten rules or things that you notice about EddieHub that aren't necessasrily clear and you would like more explanation about?

Suggestions:
- The rules when signing up to the discord are not available once you are a member, they should be available to refer to at all times.
- the rules state we use inclusive language but it should be made clearer what that means without having to find the bot config file, it can be a bit of a shock when you trigger the bot the first few times and folks don't know how to react to it
- It should also be made clear that only github links are allowed to begin with and once you become a 'regular' then you earn the right to post other links. Also an explanation of how to become a 'regular'
- Sylvian mentioned that they would like an explanation of the discord roles as it is not clear what certain roles mean and how to get them eg Among us, team Eddie, MLH-guild etc.

What other things should be made clearer? Either in the discord or in the github organisation?

**Comment by SaptarshiSarkar12** [comment]
I can not find any other rules/things that should be made clear.

**Comment by csharpforevermore** [comment]
I don't have a real understanding of what a newcomer is supposed to do.  You have this abstract introduction to your organisation, but issues are for fixing bugs, and pull requests are for feature branches of code.  It seems like your basically creating tentative links to other people's repos?

I guess in short, it would be god to tell the user what they should do and **why** on that first readme.md file when then land on EddieHub page.  I found the hardest was following the videos.  It's hard to see what criteria you are ticking off against people's repositories.

Is this a non-standard way of using GitHub to produce home pages?
